orttraining/orttraining/test/python/orttraining_test_onnxblock.py

```diff
@@ -554,6 +554,83 @@ def mse_loss(prediction, target):
         _ = ort_session.run(ort_output_names, ort_inputs)
 
 
+@pytest.mark.parametrize(
+    "block",
+    [SimpleTrainingBlockWithMSELoss, SimpleTrainingBlockWithCrossEntropyLoss, SimpleTrainingBlockWithBCEWithLogitsLoss],
+)
+def test_sgd_optimizer_composition(block):
+    # Given
+    device = "cuda"
```

Comment:
Since this is only composition test we can just use cpu device.
BTW curious are these tests only enabled in GPU pipelines?

Reply:
Seems like they only run on Test Training APIs which is on GPU.
however, we will change it to the on device training tests instead of onnxblock.